### PR TITLE
Awesome books with es6

### PR DESCRIPTION
**Pull request by tonnytech**
- Add Linters for HTML/CSS.
- Use best practices for HTML & CSS.
- Update the README file.
- Add Mark up to the .html file.
- Add basic styling to css
- Add Form to save books.
- use Local storage to save user data.
- User can add books and can remove it from the list.
- Used es6

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Awesome Books List</title>
+    <link rel="stylesheet" href="./bootstrap.min.css">
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
```
Comment (muneebulrehman) [inline index.html:11]: The main page of your application should show one section of your application. Currently, your home page shows all the sections. Kindly fix this error.
You can checkout [this](https://www.loom.com/share/90d513ae33084f34916354c4c01df732) video for more details.
image

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @teitonny , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Good design

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+});
+
+// Event: Remove a book
+document.querySelector('#book-list').addEventListener('click', (e) => {
+  //    Remove book from UI by attaching it to a method
+  UI.deleteBook(e.target);
+
+  //  Remove book from store
+  Store.removeBook(e.target.parentElement.previousElementSibling.textContent);
+});
```
Comment (muneebulrehman) [inline index.js:38]: There is an issue with the deleting of the books. Kindly check the attached video.
https://www.loom.com/share/90d513ae33084f34916354c4c01df732

File: modules/store.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+// Store Class: Handle storage - local storage
+class Store {
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
```
Comment (koachcode) [inline modules/store.js:9]: - [Optonal] Good job on completing this task, I noticed that when I add a book, I do not know if it's added or not, which affects user experience, I would suggest after submitting the form input field should be set to empty, so as to give the user a sense of submitted.

**Review by koachcode — APPROVED** [review]
Hi @teitonny , 

# STATUS: APPROVED 🟢

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**